clustering/categories.json

```diff
@@ -0,0 +1 @@
18s', 'twt017s1', 'twt017s2']}{'category_id': 1070,'minifigure_ids': ['mnn002', 'mnn001', 'mnn003', 'mnn004', 'mnn005', 'mnn006', 'mnn007', 'mnn008', 'mnn009', 'mnn010', 'mnn011', 'mnn012', 'mnn013', 'mnn014', 'mnn015']}{'category_id': 1100,'minifigure_ids': ['mk001', 'mk006', 'mk002', 'mk005', 'mk007', 'mk008', 'mk009', 'mk010', 'mk011', 'mk014', 'mk013', 'mk012', 'mk017', 'mk018', 'mk016', 'mk015', 'mk019', 'mk020', 'mk004', 'mk003', 'mk021', 'mk022', 'mk023', 'mk024', 'mk025', 'mk026a', 'mk026', 'mk027', 'mk028', 'mk029', 'mk030', 'mk031', 'mk032', 'mk033', 'mk034', 'mk035', 'mk036', 'mk037', 'mk038', 'mk039', 'mk040', 'mk041', 'mk042', 'mk043', 'mk044', 'mk045', 'mk046', 'mk047', 'mk048', 'mk049', 'mk050', 'mk051', 'mk052', 'mk053', 'mk054', 'mk055', 'mk056', 'mk057', 'mk058', 'mk059', 'mk060', 'mk061', 'mk062', 'mk063', 'mk064', 'mk065', 'mk066', 'mk067', 'mk068', 'mk069', 'mk070', 'mk071', 'mk072', 'mk073', 'mk074', 'mk075', 'mk076', 'mk077', 'mk078', 'mk079', 'mk080', 'mk081', 'mk082', 'mk083', 'mk084', 'mk085', 'mk086', 'mk093', 'mk087', 'mk088', 'mk089', 'mk090', 'mk091', 'mk092']}{'category_id': 1095,'minifigure_ids': ['mar0013', 'mar0014', 'mar0001', 'mar0015', 'mar0002', 'mar0003', 'mar0016', 'mar0017', 'mar0018', 'mar0019', 'mar0004', 'mar0020', 'mar0021', 'mar0023', 'mar0005', 'mar0006', 'mar0007', 'mar0008', 'mar0025', 'mar0026', 'mar0009', 'mar0028', 'mar0029', 'mar0030', 'mar0031', 'mar0010', 'mar0011', 'mar0032', 'mar0033', 'mar0012', 'mar0027', 'mar0022', 'mar0024', 'mar0034', 'mar0035', 'mar0036', 'mar0037', 'mar0038', 'mar0039', 'mar0040', 'mar0041', 'mar0042', 'mar0043', 'mar0044', 'mar0045', 'mar0046', 'mar0047', 'mar0048', 'mar0049', 'mar0050', 'mar0052', 'mar0051', 'mar0053', 'mar0054', 'mar0055', 'mar0056', 'mar0057', 'mar0058', 'mar0059', 'mar0060', 'mar0061', 'mar0062', 'mar0063', 'mar0064', 'mar0065', 'mar0066', 'mar0067', 'mar0068', 'mar0069', 'mar0070', 'mar0071', 'mar0072', 'mar0073', 'mar0074', 'mar0075', 'mar0076', 'mar0077', 'mar0078', 'mar0079', 'mar0080', 'mar0081', 'mar0082', 'mar0083', 'mar0084', 'mar0085', 'mar0086', 'mar0087', 'mar0088', 'mar0089', 'mar0090', 'mar0091', 'mar0092', 'mar0093', 'mar0094', 'mar0096', 'mar0097', 'mar0098', 'mar0099', 'mar0100', 'mar0102', 'mar0103', 'mar0101']}{'category_id': 1125,'minifigure_ids': ['vid001', 'vid003', 'vid004', 'vid005', 'vid015', 'vid006', 'vid007', 'vid013', 'vid012', 'vid017', 'vid009', 'vid008', 'vid011', 'vid010', 'vid018', 'vid016', 'vid014', 'vid002', 'vid004a', 'vid019', 'vid020', 'vid021', 'vid022', 'vid023', 'vid024', 'vid025', 'vid026', 'vid027', 'vid028', 'vid029', 'vid030', 'vid031', 'vid032', 'vid033', 'vid034', 'vid038', 'vid042', 'vid044', 'vid037', 'vid035', 'vid040', 'vid046', 'vid045', 'vid036', 'vid043', 'vid039', 'vid041']}{'category_id': 1154,'minifigure_ids': ['ftv001', 'ftv002', 'ftv003', 'ftv004', 'ftv005', 'ftv007', 'ftv006']}{'category_id': 1163,'minifigure_ids': ['que004', 'que005', 'que006', 'que007', 'que001', 'que002', 'que003']}
```

Comment:
`'vid041']}{'catego`
This is no valid json.
I would not upload this file to git when you can generate it